src/Player.Vm.Api/Infrastructure/BackgroundServices/CallbackBackgroundService.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright 2021 Carnegie Mellon University. All Rights Reserved.
+// Released under a MIT (SEI)-style license. See LICENSE.md in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Dataflow;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Player.Api.Client;
+using Player.Vm.Api.Data;
+using Player.Vm.Api.Domain.Services;
+using Player.Vm.Api.Features.Vms;
+using Player.Vm.Api.Infrastructure.Options;
+
+namespace Player.Vm.Api.Infrastructure.BackgroundServices
+{
+    public interface ICallbackBackgroundService
+    {
+        void AddEvent(WebhookEvent e);
+        Task<VmMap[]> CloneMaps(ViewCreated form, CancellationToken ct);
```

Comment:
Does this need to be part of the interface? I don't think anything else is calling it. It might actually make sense to move this into a separate MapService